apps/labs/posts/putting-out-the-fire.md

```diff
@@ -0,0 +1,172 @@
+on how to get started with JupyterLab and use relevant accessibility
+support, then all the work we’ve done in the software itself won’t be able
+to serve the very people it is there for.
+
+### 4. Adding relevant accessibility tests to the JupyterLab contributing workflow
+
+This is something contributors and maintainers need, though the results
+also benefit users. As grateful as I am to have a group of people who are
+taking action to make JupyterLab accessible, it isn’t enough on its own.
+We aren’t a group that can review every single PR and we may not all be
+able to devote time to this forever; tests ensure that accessibility
+remains a priority in the contributing workflow regardless of individual
+involvement. It also will help prevent current efforts from being
+overwritten by new contributions.
+
+Automated accessibility [testing has its limits](https://www.w3.org/WAI/test-evaluate/tools/selecting/)
+because you are trying to quantify an experience without getting users
+involved, but I think a first pass and a reminder to the community—especially
+the contributing community—that accessibility is something we are all
+responsible for is critical. Since accessibility isn’t yet a regular
+standard for contributions in many projects, feedback from tests might
+also be an opportunity for people who haven’t worked with accessibility
+before to start learning more.
+
+## Where we are now
+
+As I’m writing this post, our team is mostly focused on JupyterLab
+accessibility for WCAG compliance starting with the read-only type
+experience. Among many things, JupyterLab is currently missing of
+[landmarks](https://accessibility.18f.gov/landmarks/) and
+[labels](https://webaim.org/articles/label-name/) that block manual
+accessibility testing to a degree since they prevent further navigation
+and interaction. Starting here means that we are a step closer to
+users being able to accessibly read content in the interface.
+
+If you are going to take away one thing from my journey so far, I’d
+tell you to be consistently brave. Feeling anxious in the face of
+challenges and accepting areas where you don’t yet have knowledge is
+normal, but it isn’t reason to back down. Find the people that will
+collaborate with you and dive in. And when I get lost and don’t know
+what to do, I find it most helpful to put people first and remember
+who I am doing this for. Breaking the work into pieces by what users
+need can help you strategically start putting out fires.
+
+Focusing on people just for strategy isn’t all though. Be on the look
+out for my next blog where I’ll talk about what the disconnect of
+what accessibility meant to different people in our community and how
+that impacted the time and way we’ve solved issues in JupyterLab so far.
+
+___
+
+*This is part of a series of blogs around making JupyterLab more accessible. You can read the
+[whole series here](/categories/jlaba11y).*
```

Comment:
@MarsBarLee Since we don't have this category for the new Labs site yet, do we want to remove this or link to https://labs.quansight.org/blog?category=Access-centered&page=1?

Reply:
I agree on changing to the `access-centered` tag, which Gabriel changed to. Thanks!